force-app/main/default/classes/Gift.cls

```diff
@@ -82,6 +82,10 @@ public with sharing class Gift {
         delete this.dataImport;
     }
 
+    public void undeleteRecord() {
```

Comment:
Is this still needed?

Reply:
Nope, this can and will be removed shortly. Thought I had found all references to undelete, so thanks for surfacing this.